test/utils/e2e_flags.go

```diff
@@ -20,4 +20,5 @@ var (
 	TektonPipelineDeploymentLabel  = "app.kubernetes.io/part-of=tekton-pipelines"
```

Comment:
should these all be singular-ized to match the singular terms used elsewhere:

docs/TektonConfig.md
docs/README.md
https://github.com/tektoncd/operator/pull/630/files#diff-53e250d16660ac2fb7caea4b5a74b133466385a329886d4bd7ba8e1b09ae627cR92

etc